### ISSUE TITLE
Any plans on fixing the interpolation issue for MSFS? Works okay (ish) but I'm getting tired of no update on this issue and its been 3 months...

### ISSUE THREAD

**Issue by Sousaboi** [Nukem9/dlssg-to-fsr3#399]

**Comment by nivlqC** [comment]
Yeah its annoying, and hes been kinda radio silent for the past 2 months. I wouldnt expect an update any time soon, even though this is a pretty prominent issue

**Comment by Sousaboi** [comment]
Crazy how I said that this mod had crazy potential, now the author has basically dipped from it. I get it, life is busy, but if you have plans to fix an issue that many people have issues with, you would expect them to fix it rather quickly. This (imo) shouldn't take long to fix. Maybe a week or two, not 2 months.. My patience is pretty much gone now. I've waited since version .40. We are now on version .90.....

**Comment by Adriespip** [comment]
Is frame gen working for you since last update?? For me it was working good since last update, please help if somebody knows

**Comment by Sousaboi** [comment]
Like I said, it works okay (ish). I can still play it, but every now and then I’m getting a CTD and the interpolation issues are still occurring.

**Comment by xv0t** [comment]
lol you're tired? why not do it yourself then if you think it shouldn't take long to fix, that's why it's open source.
It's a mod, it's not anything official. No one is forcing you to use it.

**Comment by Sousaboi** [comment]
Right, it’s just a mod, and what I mean “it shouldn’t take long” means that it shouldn’t take 3 months for a simple answer. Hell even an update on the progress would do! Now about me doing it myself, I have absolutely no idea how this sh*t works 😆

**Comment by nivlqC** [comment]
xv0t its hard to fix something when you dont know how its made

**Comment by Maxcension** [comment]
> Right, it’s just a mod, and what I mean “it shouldn’t take long” means that it shouldn’t take 3 months for a simple answer. Hell even an update on the progress would do! Now about me doing it myself, I have absolutely no idea how this sh*t works 😆

So you don't have any idea how "this shit works" but you think that "shouldn't take long to fix"? 
This mod is free, you don't like it? You're "tired of no update"? Don't use it and don't be a jerk.

**Comment by Sousaboi** [comment]
All I’m asking for is an update whether or not the author is planning on fixing the issue. I love the mod, I plan on never removing it, as I run triple the amount of frames than before. I never intended to be a jerk by the way so I apologize for that 🙂

**Comment by TimmyP777** [comment]
Imo Nvidia backed themselves into a corner with DLSS3 marketing, and this mod exposed it. There is nothing "fancy" about frame generation when compared to ml trained up-scaling and it should NEVER have been included in DLSS "nomenclature" and just called "frame generation." I bet there are things going on in the background we dont know about, like Nvidia trying to figure out how they can bring frame generation to all RTX cards, without scorning 40 series buyers.

VR has been doing it for a long time now via spacewarp techniques like ASW or Motion Repro.

**Comment by luisguerreiro42** [comment]
> Imo Nvidia backed themselves into a corner with DLSS3 marketing, and this mod exposed it. There is nothing "fancy" about frame generation when compared to ml trained up-scaling and it should NEVER have been included in DLSS "nomenclature" and just called "frame generation." I bet there are things going on in the background we dont know about, holding this mod up, like Nvidia trying to figure out how they can bring frame generation to all RTX cards, without scorning 40 series buyers.
> 
> VR has been doing it for a long time now via spacewarp techniques like ASW or Motion Repro.

Yea people have multiple theories on this, specially since lukefz got "nuked" on his patreon because people got fed up with him

**Comment by TimmyP777** [comment]
> Yea people have multiple theories on this, specially since lukefz got "nuked" on his patreon because people got fed up with him

The way I see it, regardless of what is holding this up, and it obviously is being held up... now that we know frame generation IS decoupled from these upscaling techs, I imagine upcoming games would let us select FSR frame gen and DLSS2 for up-scaling.

If this goes in such a way where this isn't possible, and we can no longer have frame generation combined with DLSS2 up-scaling (ala what this mod does)... I'm throwing in the towel. That would be too much. Been doing this 30 years and that seems like a proper endpoint for me. Ill stick with what I have and never upgrade again, and get a console.

**Comment by ForthtuN** [comment]
Your tired of him not doing things that you want for free?

He released it. Its working really good for what it is. The bugs that is, is mostly a general FSR 3/Frame gen issue which AMD themselves will fix later on in Fsr 3.1. You can always start coding and doing it yourself instead of feeling entitled that others should do it for you

**Comment by Sousaboi** [comment]
When I say tired I mean lack of communication. Ya know if you read my previous comments I said the exact same thing. Since then I’ve understood that Nukem was “attacked” by Nvidea on here. At least he could’ve said something about it, cause it happened months ago, and I’ve just learned about what happened 2 weeks ago..

**Comment by TimmyP777** [comment]
> When I say tired I mean lack of communication. Ya know if you read my previous comments I said the exact same thing. Since then I’ve understood that Nukem was “attacked” by Nvidea on here. At least he could’ve said something about it, cause it happened months ago, and I’ve just learned about what happened 2 weeks ago..

Nobody was "attacked."

There are like... 6 places where 99% of information about this hobby is circulated, and all of these places are heavily moderated and are all already involved with Nvidia\AMD\etc for marketing etc. I personally think it goes further than this. You think these companies don't have user accounts on all these sites, posing as users to market and manipulated narratives to drive sales? 

They do, and they use it to control information. I promise you at least one person on these forums posting is one of them. Try posting on reddit pcmr or pcgaming about the specific benefits of this mod; explain that it GIVES them a free feature equal to hundreds to thousands of dollars worth of an upgrade... what do you see? Downvotes, which orphans the comment so nobody can see. These downvotes come from the narratives being propagated by PAID associates of companies and the people that follow (everyone, everyone is brainwashed- vram, broken games, architecture, "fake frames," etc), and they are moderated 24/7. Thus nobody learns, and the information is held back. Its not just the pc community either, its everywhere.

They might also be a paid 3rd party entity similar to those "buy upvotes" sites, where you could see pictures of hundreds\thousands of machines each with unique accounts for upvoting, instead or in addition to the companies themselves.

My question is how? How do they remove posts, or get posts removed, in minutes? Every single time in my case. Try it on reddit, make sure you log out and check if the comment is there. Shadowbanned\muted accounts will show you your posts as if they are "live" but only when you are logged in.
